environments/environment.yml

```diff
@@ -6,5 +6,6 @@ channels:
 dependencies:
   - python=3
   - fastp=0.23.2
-  - tb-profiler=6.2.0
+  - tb-profiler=6.2.1
   - qualimap=2.2.2d
+  - pathogen-profiler=4.2.0
```

Comment:
I don't think we need to pin pathogen-profiler here. It's pinned in the biocond recipe for tb-profiler:

https://github.com/bioconda/bioconda-recipes/blob/d9b5822dee1b488a893ee7c4f6188983ef89e026/recipes/tb-profiler/meta.yaml#L31